### ISSUE TITLE
investigate NAs of usertype

### ISSUE THREAD

**Issue by vegahi** [vegahi/thesis#13]
check if there's missing information in the raw data files

**Comment by vegahi** [comment]
I ran the following query and found that the missing data of usertype only exits in three months.

````
SELECT year(Start_Time) as the_year, month(Start_Time) as the_month, User_Type, sum(1) as N
FROM citibike.trips
WHERE User_Type NOT IN ('Subscriber', 'Customer')
GROUP BY the_month, the_year, User_Type;
````
The result looks like 
````
the_year,the_month,User_Type,N
2016,10,,17949
2016,11,,12534
2016,12,,5388
````
The sums of the three add up to 35871, which is the same as the total number of missing ones. I checked the original data and the rows with missing data are not exactly consecutive. 

The total number of observations for 201610 is 1573872, which means that the missing data is around 0.011. Similarly, 201611 has  0.010 and 201612 has 0.007.
